app/lib/common/widget/FaqListItem.dart

```diff
@@ -0,0 +1,319 @@
+              },
+            ),
+          );
+        },
+        child: Column(
+          children: <Widget>[
+            // ListTile(
+            // title: Text(
+            //   widget.faq.title(),
+            //   style: optionStyle,
+            //   maxLines: 2,
+            // ),
+            // trailing: Icon(Icons.turned_in_outlined, color: Colors.grey),
+            // subtitle:
+            Column(
+              children: <Widget>[
+                Container(
+                  child: Card(
+                    child: Padding(
+                      padding: EdgeInsets.fromLTRB(8.0, 8.0, 8.0, 8.0),
+                      child: Column(
+                        children: [
+                          Padding(
+                            padding: EdgeInsets.fromLTRB(12.0, 4.0, 12.0, 4.0),
+                            child: Row(
+                              children: [
+                                Flexible(
+                                  child: Text(
+                                    widget.faq.title(),
+                                    style: GoogleFonts.montserrat(
+                                      color: Colors.white,
+                                      fontSize: 16,
+                                      fontWeight: FontWeight.w600,
+                                    ),
+                                  ),
+                                ),
+                                // new Spacer(),
+                                GestureDetector(
+                                  onTap: () => {
+                                    Fluttertoast.showToast(
+                                      msg: 'Bookmark icon tapped',
+                                      toastLength: Toast.LENGTH_SHORT,
+                                    ),
+                                  },
+                                  child: Image.asset(
+                                    'assets/images/bookmark.png',
+                                  ),
+                                )
+                              ],
+                            ),
+                          ),
+                          Padding(
+                            padding: EdgeInsets.only(top: 12.0, bottom: 12.0),
+                            child: Row(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: [
+                                Container(
+                                  height: 40.0,
+                                  padding: EdgeInsets.all(8.0),
+                                  margin: EdgeInsets.only(left: 12.0),
+                                  decoration: BoxDecoration(
+                                    borderRadius: BorderRadius.circular(10),
+                                    border: Border.all(
+                                      color: Color(0XFFFFFFFFF),
```

Comment:
Why are there static color definitions in this? We are using the Material Theme system, please use that instead of static colors. This goes for the entire view.